src/host_toolchains.py

```diff
@@ -94,7 +106,10 @@ def SetUpVSEnv(outdir):
 def CopyDlls(dir, configuration):
   """Copy MSVS Runtime dlls into a build directory"""
   proc.check_call([VS_TOOLCHAIN, 'copy_dlls', dir, configuration, 'x64'])
-
+  # LLD needs also concrt140.dll, which the Chromium copy_dlls doesn't include.
+  for dll in glob.glob(os.path.join(GetRuntimeDir(), 'concrt140*.dll')):
+    print 'Copying %s to %s' % (dll, dir)
+    shutil.copy2(dll, dir)
```

Comment:
I wonder why?  Presumably chrome is using lld so they should need all the same dlls?

Reply:
Chrome may or may not be using lld to link (clang works with gnu ld, gold and link.exe) but this dll-copying is for distribution, i.e. running on a machine other than a build machine. So Chrome itself doesn't necessarily need the same dlls.